Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h

```diff
@@ -20,6 +21,96 @@ namespace phase1PixelTopology {
 
   constexpr uint32_t numPixsInModule = uint32_t(numRowsInModule) * uint32_t(numColsInModule);
 
+  constexpr uint32_t numberOfModules = 1856;
+  constexpr uint32_t numberOfLayers = 10;
+  constexpr uint32_t layerStart[numberOfLayers + 1] = {0,
```

Comment:
I think it could, but it would require changes in downstream code to get the underlying pointer.
A simpler change is to keep it as a C array, and use `std::size(layerStart)` below.

Reply:
See https://github.com/cms-patatrack/cmssw/pull/593/commits/7ae9dde4393cdff2913d83915fff9aae12b2cf67 .

Reply:
std array is (was?) not cuda friendly (missing constexpr, cannot be allocated because of constructors...